src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -114,8 +114,9 @@ protected bool InitializeFileWatcherIfEnabled()
 
             string invocationId = functionExecutionContext.InvocationId.ToString();
 
-            FunctionStartedEvent startedEvent = new FunctionStartedEvent(functionExecutionContext.InvocationId, Metadata);
+            var startedEvent = new FunctionStartedEvent(functionExecutionContext.InvocationId, Metadata);
             _metrics.BeginEvent(startedEvent);
+            var invokeLatencyEvent = _metrics.BeginEvent(MetricEventNames.FunctionInvokeLatency);
```

Comment:
We already have a `function.invoke` count event. I could have removed that in favor of this new `function.invoke.latency` event, but it doesn't hurt to have both.